release/reorder-tags.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# for some reason, github always puts our enterprise release notes above our open source
+# release notes on the releases page. We always want the open source release notes to be on top.
+# The hack we've found for this is to force-push yesterday's date into the enterprise release tag.
```

Comment:
This trick may not work if we ever need to have 2 releases in 1 day.
We'd probably end up with something like:
1. :heavy_check_mark: latest oss
2. :x: previous oss
3. :x: latest ee
4. :heavy_check_mark: previous ee

Reply:
We've had situations in the past where we released three or even four releases in a day (backporting a hotfix).
My manual approach was to apply this hack to each individual EE version.

And to confirm - yes, I did run into this problem.
There isn't a shared opinion on how to order multiple releases across versions, though. Some people think it's best to have OSS-EE grouped in pairs **per version**. Others are ok with OSS-OSS, followed by EE-EE.

We should probably decide which one to follow and stick with it.

Reply:
yeah, if we have multiple releases in the same day, we'll probably have to reorder manually, but it happens so infrequently, I don't know that it's worth building automation for it.

Reply:
Don't bother with that.
This already solves most of our pain points from the past.